docker-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
       volumes:
         - "kafka_data:/bitnami"
         - ./scripts/create-all-topics.sh:/usr/bin/create-topics.sh
-      command: "sh -c './run.sh & /usr/bin/create-topics.sh && tail -f /dev/null'"
+      command: "sh -c './run.sh && tail -f /dev/null'"
```

Comment:
Why did we remove the call to create-topic.sh? It creates KAFKA topics if they haven't been created before. It's an important step in "single-command docker run"

Reply:
We don't. It's a local change from @Ris-code machine. So I have asked to revert it back.